src/node_contextify.cc

```diff
@@ -950,86 +765,55 @@ class ContextifyScript : public BaseObject {
   }
 
 
-  // args: [options]
   static void RunInThisContext(const FunctionCallbackInfo<Value>& args) {
     Environment* env = Environment::GetCurrent(args);
 
-    // Assemble arguments
-    TryCatch try_catch(args.GetIsolate());
-    Maybe<int64_t> maybe_timeout = GetTimeoutArg(env, args[0]);
-    Maybe<bool> maybe_display_errors = GetDisplayErrorsArg(env, args[0]);
-    Maybe<bool> maybe_break_on_sigint = GetBreakOnSigintArg(env, args[0]);
-    if (try_catch.HasCaught()) {
-      try_catch.ReThrow();
-      return;
-    }
+    CHECK_EQ(args.Length(), 3);
 
-    int64_t timeout = maybe_timeout.ToChecked();
-    bool display_errors = maybe_display_errors.ToChecked();
-    bool break_on_sigint = maybe_break_on_sigint.ToChecked();
+    CHECK(args[0]->IsNumber());
+    int64_t timeout = args[0].As<Integer>()->Value();
```

Comment:
The check doesn't match the cast.  It's not exactly wrong because `Integer::Value()` does the right thing regardless but stil..

You could instead use `IntegerValue()` (but please use the overload that takes a `Local<Context>`.)